lib/player.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+
+# class Player
+class Player
+  include UI
+
+  attr_reader :sign
+  def initialize(sign)
+    @sign = sign
+  end
+
+  def make_move(board)
```

Comment:
As mentioned before, all public methods should be tested. 
Create player_spec.rb file to test this method.

Thanks 😄

Reply:
The change for this method has been done